src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -759,7 +793,10 @@ private IUserSecurityStampStore<TUser> GetSecurityStore()
             CancellationToken cancellationToken = default(CancellationToken))
         {
             ThrowIfDisposed();
-            return await GenerateUserTokenAsync(user, Options.PasswordResetTokenProvider, "ResetPassword", cancellationToken);
+            var token = await GenerateUserTokenAsync(user, Options.PasswordResetTokenProvider, "ResetPassword", cancellationToken);
+            await LogResultAsync(IdentityResult.Success, user);
```

Comment:
Actually looks like there should be the following log overloads potentially

Log(bool, ...)
Log(...)
Log(IdentityResult, ...)

rather than LogResult